tests/unit/usecases/test_token_balances_api.py

```diff
@@ -0,0 +1,42 @@
+import json
```

Comment:
I created this test to match the coverage so it is unrelated to the core matter of the PR.
I chose the token balances api usecase since it had 0% coverage, now it has 100%.